docs/changelog.md

```diff
@@ -148,6 +148,12 @@ permalink: /changelog/
     * Upgrade glean_parser to v1.12.0.
     * Implement the deletion request ping in Glean.
 
+* **service-sync-logins**
```

Comment:
Drive-by comment: this changelog entry probably needs to go to the latest release.

Reply:
Thanks! I guess this had gone more stale than I thought :-)